src/api/aux.cpp

```diff
@@ -22,6 +22,9 @@ void parse_json_file(std::vector<Token> &tokens, std::vector<Order> &orders, std
     
     for (auto &i: json_file["tokens"].items()) {
```

Comment:
Suggestion: rename "i" as "token", or "t" or something that is easy to remember.

Reply:
Done